src/scss/layout/_header.scss

```diff
@@ -2,26 +2,47 @@
 
 .site-header {
   // header section
+  position: relative;
   display: flex;
   justify-content: space-between;
-
+  height: rem(80px);
+  
+  &::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: calc((100vw - 100%) / -2 - 10px);
```

Comment:
This is weird, but on my computer, the -10px part of this calculation actually makes the line not go the full width of the screen. The same is true for the methodology section. Can you go ahead and remove it for now (from both sections), and I'll review this later for compatibility?

<img width="641" alt="Screen Shot 2019-11-06 at 10 30 03 AM" src="https://user-images.githubusercontent.com/1896496/68312126-968f2980-0080-11ea-94da-3c3559bf0d9d.png">